packages/stark-ui/src/modules/date-picker/components/date-picker.component.html

```diff
@@ -1,17 +1,28 @@
 <mat-form-field [id]="pickerId">
 	<input
 		matInput
-		[disabled]="isDisabled"
 		[id]="pickerId + '-input'"
 		[matDatepicker]="picker"
+		[value]="value"
+		(dateInput)="onDateInput($event)"
+		(dateChange)="onDateChange($event)"
 		[placeholder]="label | translate"
 		[matDatepickerFilter]="dateFilterFnWrapper"
 		[min]="minDate"
+		[disabled]="disabled"
 		[max]="maxDate"
 		[name]="pickerName"
-		[value]="date"
-		(dateChange)="onDateChange($event)"
+		[required]="required"
+		(focus)="onFocus($event)"
```

Comment:
I think the `onFocus()` method can be simplified if you pass directly the input element by doing this: 

```html
<input matInput
     #pickerInput="matDatepickerInput"   <---- this will export this input as "pickerInput"
     ...
     (focus)="onFocus(pickerInput)"  <---- so in the onFocus() you already get the input element
     
```